documentation/assemblies/operators/assembly-using-multiple-cluster-operators.adoc

```diff
@@ -0,0 +1,24 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-the-cluster-operator.adoc
+
+[id='assembly-using-multiple-cluster-operator-replicas-{context}']
+= Running multiple Cluster Operator replicas with leader election
+
+[role="_abstract"]
+Enable xref:ref-operator-cluster-leader-election-{context}[leader election] in the Cluster Operator configuration to run multiple parallel replicas of the Cluster Operator.
```

Comment:
Should we add that its enabled by default in the installation files?

Reply:
Changed to:

> The default Cluster Operator configuration enables [leader election]. 
> Use leader election to run multiple parallel replicas of the Cluster Operator.